docs/presentation/Content/PropertyCategorySpecification.md

```diff
@@ -1,56 +1,117 @@
-| `description` | No        | `string`                      | `""`    | Extensive description of the category.                                                                                                       |
-| `priority`    | No        | `number`                      | `1000`  | Priority of the category. Higher priority categories are displayed on top.                                                                   |
-| `autoExpand`  | No        | `boolean`                     | `false` | Should this category be auto-expanded.                                                                                                       |
-
-## Examples
-
-```JSON
-{
-  "ruleType": "ContentModifier",
-  "class": {"schemaName": "BisCore", "classNames": ["MyElementClass"]},
-  "propertyCategories": [{
-    "id": "my_custom_root_category",
-    "label": "My Custom Root Category"
-  }, {
-    "id": "my_custom_child_category",
-    "parentId": "my_custom_root_category",
-    "label": "My Custom Child Category"
-  }],
-  "propertyOverrides": [{
-    "name": "MyProperty1",
-    "categoryId": {
-      "type": "Id",
-      "id": "my_custom_root_category"
-    }
-  }],
-  "relatedProperties": [{
-    "propertiesSource": {
-      "relationship": {"schemaName": "BisCore", "className": "SomeRelationship"},
-      "direction": "Forward",
-    },
-    "properties": [{
-      "name": "RelatedProperty2",
-      "categoryId": {
-        "type": "Id",
-        "id": "my_custom_child_category"
-      }
-    }]
-  }]
-}
+| Name                                    | Required? | Type                                                  | Default     |
+| --------------------------------------- | --------- | ----------------------------------------------------- | ----------- |
+| [`id`](#attribute-id)                   | Yes       | `string`                                              |             |
+| [`parentId`](#attribute-parentid)       | No        | `string \| CategoryIdentifier`                        |             |
+| [`label`](#attribute-label)             | Yes       | `string`                                              |             |
+| [`description`](#attribute-description) | No        | `string`                                              | `""`        |
+| [`priority`](#attribute-priority)       | No        | `number`                                              | `1000`      |
+| [`autoExpand`](#attribute-autoexpand)   | No        | `boolean`                                             | `false`     |
+| [`renderer`](#attribute-renderer)       | No        | [`RendererSpecification`](./RendererSpecification.md) | `undefined` |
+
+### Attribute: `id`
+
+Category identifier used to reference the category definition from property overrides or other category definitions. The identifier has to be unique at the
+scope of it's definition.
+
```

Comment:
It does not have even a test example, was there a reason for this decision or just miseed?

Reply:
All other examples also use this attribute, so I didn't see a point to repeat the same stuff. Do you think it's not clear? If yes, we could add a link to the example of `label` attribute that's right below this one.

Reply:
Maybe not so much as 'unclear', but opening up the page, at first sight this section looks lacking in documentation. You are right that you can just loot at the examples below. This might be a matter of preference, I personally made separate examples. I leave the decision up to you.

Reply:
I'd leave it as is. @saskliutas or @roluk, have a preference?

Reply:
The description suggests that the id has to be unique only within a certain bound. You could show an example where two categories have the same id or demonstrate how a referenced id can be unreachable from some specification, if that's even a thing that could happen.

Reply:
Added an example and changed the description to make it clear where the uniqueness has to be kept.